.config/tsaoptions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Runtime\\Reliability\\Docs",
```

Comment:
Any reason docs is the path we'd use?

Reply:
I had initially taken what was there in `sdl-tsa-vars.config`, but modifying it to `DevDiv\\NET Libraries` as recommended by https://github.com/dotnet/runtime/pull/76751#discussion_r1006045430